### DISCUSSION TITLE
缺少cefwrapper的lib库

### DISCUSSION THREAD

**Discussion by MikeXuQ** [netease-im/NIM_Duilib_Framework#485] [General]
将duilib加入到我的工程中，设置好各种宏定义，编译成功后显示很多函数链接失败，这些函数应该是存储在对应的lib文件吧？但如果我按照快速上手的步骤执行，则不会报错，想请教一下大佬原因是啥呢？

**Comment by nmgwddj** [comment]
哪些函数无法链接到呢？通过您现有的描述我很难帮到您。

**Reply by MikeXuQ** [reply]
在cefwrapper中没有实现的函数都会链接失败，比如整个cef_web_plugin_capi.h中涉及到的函数；虽然这个问题我通过换了个libcef库解决了，但还是很好奇原因。
